### ISSUE TITLE
Enemies won't spawn in v1.1.34.

### ISSUE THREAD

**Issue by stb06** [DestroyedClone/AncientScepter#49]
It also seems to always put me on Titanic Plains first, and without the drop pod or controls. The camera is also in the ground. This might be related to using it with Wayfaring2. Furthermore, it seems to at least semi-disable StageAesthetic.

**Comment by DestroyedClone** [comment]
Could you post your bepinex log with it happening?

**Comment by stb06** [comment]
[Log.txt](https://github.com/user-attachments/files/17008565/Log.txt)

**Comment by MGJerry** [comment]
I don't think either this mod is causing the the first 3 issues since my runs with the bois since 2 days ago haven't encounter these, so it's probably caused by mods that change enemies spawning (for 1st issue), stage order (for 2nd), drop pod and player spawn logic (for 3nd).

As for StageAesthetic, it hasn't been updated since SOTS so there has been visual bugs here and there, I have seen a fair few myself.

**Comment by stb06** [comment]
It only happens when I enable the mod though, so I know it has to do with incompatibilities somehow.

**Comment by MGJerry** [comment]
Yeah, there's definitely something conflicting with this mod. If you're using mod managers like r2modman or tmm, can you export your current modpack code here? I using all the mods from the log together and can't replicate it so far.

**Comment by stb06** [comment]
0191ff2f-b76d-0f3c-d283-9a3810133a40

**Comment by MGJerry** [comment]
Hmm, I started about 10+ runs now, all at Titanic Plains and with night time/dark stage aesthetic... without this mod enabled. Might look around to see what's causing it.

**Comment by stb06** [comment]
[LogOutput.log](https://github.com/user-attachments/files/17055602/LogOutput.log)
I think I might have sent the wrong log. Here's another one with the broken camera.

**Comment by MGJerry** [comment]
Ah, I forgot to edit the previous post. Good news, I was able to replicate the no monster (and interactable) part pretty consistently. It seems to happen on the 2nd run onwards every time for me. It's a wild guess but the issue seems to be this part
![image](https://github.com/user-attachments/assets/934e4b3b-20d2-4e6b-9e1f-6efe2d6b2b95)

**Comment by DestroyedClone** [comment]
> Ah, I forgot to edit the previous post. Good news, I was able to replicate the no monster (and interactable) part pretty consistently. It seems to happen on the 2nd run onwards every time for me. It's a wild guess but the issue seems to be this part ![image](https://private-user-images.githubusercontent.com/85227828/369000912-934e4b3b-20d2-4e6b-9e1f-6efe2d6b2b95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3NzIyOTMsIm5iZiI6MTcyNjc3MTk5MywicGF0aCI6Ii84NTIyNzgyOC8zNjkwMDA5MTItOTM0ZTRiM2ItMjBkMi00ZTZiLTllMWYtNmVmZTJkNmIyYjk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE5VDE4NTMxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxYzdkNWZhOTQ5ZjhjZTdlM2Q1OGI0NTI0ZTc4NjcxN2I2NGMxZjg1ZDQ1ZWFkMTc2MDE0ZmNkZmM5ODkyY2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UfAyf_TISK-FiTCD7SpK1v64bEE7GEipNnOQs7iu4-Y)
> 
> Edit: And ye, the camera part does happens, but pretty rare it seems

https://github.com/DestroyedClone/AncientScepter/blob/9dc3ac1aa515373ac9ffab3b85c449bfd905d88c/AncientScepter/AncientScepterItem.cs#L668C1-L693C10
The lines of code that that error corresponds to is weird, because that code should only be running once on the first run, then never again for the rest of the session.
https://github.com/DestroyedClone/AncientScepter/blob/9dc3ac1aa515373ac9ffab3b85c449bfd905d88c/AncientScepter/AncientScepterItem.cs#L669
this is running more than once, which is why that error is showing up, but it shouldn't

> [LogOutput.log](https://github.com/user-attachments/files/17055602/LogOutput.log) I think I might have sent the wrong log. Here's another one with the broken camera.

I'm seeing a lot of errors in this log which may or may not be contributing to the issue, I see SurvivorsPlus, a deprecated mod, enabled in the pack with a MissingFieldException. There are also other mod errors that don't belong to this mod

**Comment by DestroyedClone** [comment]
Closing due to potential causes being other mods (MissingFieldExceptions) and no other reports of this issue. Can be re-opened if it happens again.
